WordPress/src/main/res/layout/navbar_item_old.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_centerInParent="true"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <FrameLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/nav_icon"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_gravity="center"
+            android:contentDescription="@string/tabbar_accessibility_label_my_site"
+            android:tint="@drawable/nav_bar_button_selector"
+            tools:src="@drawable/ic_create_white_24dp"/>
+
+        <View
+            android:id="@+id/badge"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_gravity="top"
+            android:background="@drawable/bg_oval_accent_stroke_white"
+            android:visibility="gone"
+            tools:visibility="visible"/>
+
+    </FrameLayout>
+
+    <TextView
+        android:id="@+id/nav_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:duplicateParentState="true"
+        android:singleLine="true"
+        android:textColor="@color/primary_40"
+        android:textSize="12sp"
```

Comment:
should we maybe extract the dimensions from this file?

Reply:
Yep makes sense. I actually changed the navbar_item.xml file and kept this one as is since we are going to remove this one once merged. Changes here 525dd4a4781231c0b6562ba15504c23acb35ca66